src/Centreon/Domain/HostConfiguration/HostConfigurationService.php

```diff
@@ -688,4 +688,17 @@ public function findHostByName(string $hostName): ?Host
             throw new HostConfigurationException(_('Error while searching for the host'), 0, $ex);
         }
     }
+
+    /**
+     * {@inheritDoc}
+     * @throws HostConfigurationException
+     */
+    public function findHostTemplateById(int $hostTemplateId): ?Host
```

Comment:
```suggestion
    public function findHostTemplate(int $hostTemplateId): ?Host
```
While not precising "By..." it is implicitly ById